solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -717,26 +718,35 @@ contract RandomBeacon is Ownable {
 
     /// @notice Creates a new relay entry.
     /// @param entry Group BLS signature over the previous entry.
-    function submitRelayEntry(bytes calldata entry) external {
+    /// @param groupMembers Group member ids that participated in dkg (excluding IA/DQ).
+    function submitRelayEntry(
```

Comment:
Given that the cost of `submitRelayEntry` increased by `21k`, can we add an alternative `submitRelayEntry` function that accepts no `groupMembers` and can be called only before the soft timeout? 

Let's see what is overall gas cost of this change to the system.

Reply:
Yes, I've actually started doing it as part of this PR in commit 61977c2 but reverted in beb7377 since there were already a lot of changes in the current PR. Any objections if I open a separate PR for `submitRelayEntry` split?

Reply:
Is it that much work? If we want it to be a separate PR, let's do not merge this one until we have full clarity about the costs. That is, merge two PRs one after another.

Reply:
Draft PR is [here](https://github.com/keep-network/keep-core/pull/2772). I still need to modify a couple of tests there, but we can compare the gas usage.